### ISSUE TITLE
Disabling MTVU Speed Hack in GameDB

### ISSUE THREAD

**Issue by Catarax** [PCSX2/pcsx2#1257]
Hello,
Does it's possible to disable MTVU speed hack with GameDB ?
It would be nice in order to fix some games and being able to enjoy this option without having to switch too much.

Thanks!

**Comment by refractionpcsx2** [comment]
Doesn't look like it, you'll just have to remember to turn it off for some games :(

It could possibly be added though.

**Comment by bositman** [comment]
Well MTVU is off by default so in the same sense you should have GameDB options to disable any speed hack that causes issues (and that's a lot :P ) I think it's beyond the scope of good per game settings (like round modes or clamping)

**Comment by refractionpcsx2** [comment]
@bositman maybe, however we do have a list of games which don't work with MTVU, it could potentially reduce support traffic by forcing it off for specific games we know definitely don't work with it on.  I know I like to have it on if I can due to it's huge performance boost, but I do forget which games don't like it :P Certainly would be more user friendly.

**Comment by jcdenton2k** [comment]
I'd love to see a comprehensive full update of the GameDB and Widescreen hacks for 2016 since many of the workarounds posted on the PCSX2 forums haven't yet been integrated into the newest GameDB and widescreen patch archives in the Git releases.

**Comment by willkuer** [comment]
Maybe you collect the missing gamedb hacks in a separate issue. I don't think that there are many missing entries.

The mtvu hack disabling will be available only after ssakash finished #1266. Unfortunately there are some annoying unresolved obstacles due to threading.so maybe it will still take some time.

Regarding widescreen hacks they were updated prior to the release of pcsx2 1.4 and are thus quite updated. Update of widescreen hacks normally only happens after devina (something like the overviewer/host for ws hacks) and a dev (often avih) decide that the updates are sufficient. Was there so much progress that is missing in releases?

**Comment by jcdenton2k** [comment]
There are about ~6 or so missing widescreen hacks from the official latest git release.
I'll make a post about it later on so they can be properly added to the next release.

**Comment by lightningterror** [comment]
I'll close this since we have a compatibility list #1669, plus the speedhack is off by default.

Atomic and Akash were working at one point, it's not high priority tho.
